docs/modules/device-lab-management/pages/android-devices/prepare-android-device.adoc

```diff
@@ -0,0 +1,144 @@
+= Prepare an Android device for hosting
+:navtitle: Prepare an Android device
+
+Learn how to prepare an Android device to be utilized with Kobiton 4.0 and later.
+
+Different models of devices and versions of Android may have different layout for settings and options. Refer to the device's manual if you cannot find the options mentioned in this document.
+
+== Download Google Chrome
+
+Download and install Google Chrome from the Google Play Store onto the device.
+
+== Enable USB debugging
+
+From the Home screen, swipe up to show all apps. Open **Settings**.
+
+image:device-lab-management:device-lab-management-android-prepare-usb-debugging-settings.png[]
```

Comment:
@Hikkigaya For all images in this PR that you added:
- change from `image:` to `image::` so it is center-aligned
- provide the width and alt information. Example:

`device-lab-management-android-prepare-usb-debugging-settings.png[width=400,alt="The Settings app in the home screen]

Reply:
- All image: changes to image::
- Width and Alt text added for all images added in this PR.